shared/analytics_tracking/noop.py

```diff
@@ -0,0 +1,14 @@
+import logging
+
+from shared.analytics_tracking.base import BaseAnalyticsTool
+
+log = logging.getLogger("__name__")
+
+
+class NoopTool(BaseAnalyticsTool):
+    @classmethod
+    def is_enabled(cls):
+        return False
+
+    def track_event(self, user_id, event_name, *, is_enterprise, event_data={}):
+        log.warning("Analytics tool is not enabled. Please check your configuration.")
```

Comment:
I think this warning should be moved to `get_analytics_tool`. Otherwise it might spam users that opt to not have the analytics set (as we discussed, they might not care / not need it).

Then this method can do nothing

Reply:
yup you're right